# James Michael Melott

I work remotely for MD Anderson Cancer Center in Houston Texas as a Senior Systems Analyst where I do software design, development and testing.

[Linked in](https://www.linkedin.com/in/james-melott-b60a8913/)

[Work background (Resume)](https://jamesmelott.com/resume/Resume_of_James_Michael_Melott.html)

[Books I've published for sale on Amazon](https://jamesmelott.com/products/index.html)

## Software repositories and websites with I am involved (Source code contributions and/or issue reporting and/or documentation)

### Pathways Web and Pathways Browser
- [Info](https://bioinformatics.mdanderson.org/public-software/pathwaysweb/) - 
[PathwaysWeb: a gene pathways API with directional interactions, expanded gene ontology, and versioning](https://pubmed.ncbi.nlm.nih.gov/26400039/)
(https://pubmed.ncbi.nlm.nih.gov/32269754/) 
- [PathwaysBrowser website](https://bioinformatics.mdanderson.org/PathwaysBrowser)
- [PathwaysWeb REST-like API](http://bioinformatics.mdanderson.org/PathwaysWeb/versions) 

### OmicPioneer-sc and Next Generation Clustered Heatmaps (NG-CHM)
- [Next Generation Clustered Heat Maps](https://bioinformatics.mdanderson.org/public-software/ngchm) 
- [Blog](https://www.ngchm.net/) 
- [Related Youtube Videos](https://www.youtube.com/channel/UCADGir2q8IaI9cGQuzjSL9w) 
- [OmicPioneer-sc Publication](https://www.biorxiv.org/content/10.1101/2020.10.31.363580v1.full.pdf) 

Related Github Repositories: 
- NGCHM Viewer [Github Repo](https://github.com/MD-Anderson-Bioinformatics/NG-CHM)
- GUI Heatmap Builder [Github Repo](https://github.com/MD-Anderson-Bioinformatics/NG-CHM_GUI_BUILDER) -- [Live Software](https://build.ngchm.net/NGCHM-web-builder/)
- Galaxy NGCHM Builder [Github Repo](https://github.com/MD-Anderson-Bioinformatics/NG-CHM_Galaxy) -- [Galaxy Toolshed](https://toolshed.g2.bx.psu.edu/repository?repository_id=858b27481f992d38) [Galaxy Docker Image](https://hub.docker.com/r/ngchm/ngchm-galaxy/) -- [A Galaxy Implementation of Next-Generation Clustered Heatmaps for Interactive Exploration of Molecular Profiling Data](https://cancerres.aacrjournals.org/content/77/21/e23)
- GenePattern NGCHM Builder and Viewer [Basic NGCHM Builder Github Repo](https://github.com/MD-Anderson-Bioinformatics/GenePattern_NGCHM_BasicBuilder) -- [GenePattern NGCHM Viewer](https://github.com/MD-Anderson-Bioinformatics/GenePattern_NGCHM_Viewer)
- NGCHM-R : R Package for building NGCHMs  [Github Repo](https://github.com/MD-Anderson-Bioinformatics/NGCHM-R)
- NGCHM RStudio Docker container [Github Repo](https://github.com/MD-Anderson-Bioinformatics/rstudio-ngchm)
- PathwaysMapper plugin for NGCHM Viewer [Github Repo](https://github.com/MD-Anderson-Bioinformatics/pathway-mapper)
- 2D ScatterPlot plugin for NGCHM Viewer [Github Repo](https://github.com/MD-Anderson-Bioinformatics/ScatterPlotPlugin)
- 3d ScatterPlot plugin for NGCHM Viewer [Github Repo](https://github.com/MD-Anderson-Bioinformatics/ScatterPlotPlugin3D)

### Reverse-Phase Protein Array Super Position and Concentration Evaluation
- [RPPASPACE R package on CRAN](https://cran.r-project.org/web/packages/RPPASPACE/index.html) 
- [Github Repo](https://github.com/MD-Anderson-Bioinformatics/rppaspace)
- [Guide to Use of RPPASPACE](https://cran.r-project.org/web/packages/RPPASPACE/vignettes/Guide_to_RPPASPACE.pdf)
- Predecessor package SuperCurve [RForge Packages](https://r-forge.r-project.org/R/?group_id=1899) - [Info](https://bioinformatics.mdanderson.org/public-software/supercurve/)

### Batch Effects and Standardized Data
- MBatch and MBatchUtils R Packages and MBatch Stand-Alone Docker Image [Github Repo](https://github.com/MD-Anderson-Bioinformatics/BatchEffectsPackage) 
- [MBatch Omics Browser](https://bioinformatics.mdanderson.org/MQA/) 
- [Info](https://bioinformatics.mdanderson.org/public-software/tcga-batch-effects/)
- Standardized Data [Github Repo](https://github.com/MD-Anderson-Bioinformatics/StandardizedData)
- DataAPI [Github Repo](https://github.com/MD-Anderson-Bioinformatics/DataAPI)

### Ideogram Viewer 
- An interactive tool for viewing the location and annotation of genes on human chromosomes.
- [Ideogram Viewer Website](https://bioinformatics.mdanderson.org/ideogramviewer/Ideogram.html?genelist1=ERBB2,TP53,BRCA1,PTEN,AGL&genelist2=TP53,MDM2,ERBB2,ERBB4&mirlist1=hsa-let-7a-2,hsa-let-7c,hsa-let-7e&mirlist2=hsa-let-7c,hsa-let-7a-2) 
- [Info](https://bioinformatics.mdanderson.org/public-software/ideogramviewer/)

### The Cancer Proteome Atlas (TCPA)
- A comprehensive resource for accessing, visualizing, and analyzing cancer functional proteomics.
- [TCPA Website](https://tcpaportal.org/tcpa/) 
- [Info](https://bioinformatics.mdanderson.org/public-software/tcpa/) 
- [TCPA: a resource for cancer functional proteomics data](http://dx.doi.org/10.1038/nmeth.2650)

### TCGA SpliceSeq
- A tool for investigating alternative mRNA splicing in next generation mRNA sequence data
- [SpliceSeq Website](https://bioinformatics.mdanderson.org/TCGASpliceSeq/)
- [Github Repo](https://github.com/MD-Anderson-Bioinformatics/SpliceSeq)
- [SpliceSeq: a resource for analysis and visualization of RNA-Seq data on alternative splicing and its functional impacts](https://academic.oup.com/bioinformatics/article/28/18/2385/253789)
- [Info](https://bioinformatics.mdanderson.org/public-software/spliceseq/) 

### SOS Notebook
- Script of Scripts : An interactive multi-language data analysis environment 
- [SoS Notebook: an interactive multi-language data analysis environment](https://academic.oup.com/bioinformatics/article/34/21/3768/5001386)
- [Github Repo](https://github.com/vatlab/SoS)
- [Info](https://vatlab.github.io/sos-docs/)

### The Atlas of non-coding RNA in Cancer (TANRIC)
- [TANRIC Website](https://ibl.mdanderson.org/tanric/_design/basic/main.html)
- [TANRIC: An Interactive Open Platform to Explore the Function of lncRNAs in Cancer](https://cancerres.aacrjournals.org/content/75/18/3728)
- [Info](https://bioinformatics.mdanderson.org/public-software/tanric/)

### TansVar
- TransVar is a multi-way annotator for genetic elements and genetic variations.
- [TransVar Website](https://bioinformatics.mdanderson.org/transvar/)
- [User Guide](https://transvar.readthedocs.io/en/latest/)
- [Info](https://bioinformatics.mdanderson.org/public-software/transvar/)
- [TransVar: a multilevel variant annotator for precision genomics](https://www.nature.com/articles/nmeth.3622)
